src/mock.ts

```diff
@@ -51,26 +61,26 @@ function addMockFunctionsToSchema({ schema, mocks = {}, preserveResolvers = fals
     }
   });
 
-  const defaultMockMap = new Map();
+  const defaultMockMap: Map<string, IMockFn> = new Map();
```

Comment:
I think the mock definition can also just be a value, it doesn't have to be a function, so IMockFn alone isn't sufficient.